sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -869,8 +859,7 @@ def download_manifest(self, repository, tag_or_digest, **kwargs):
         return DownloadManifestResult(digest=digest, data=manifest_stream, manifest=manifest)
 
     @distributed_trace
-    def download_blob(self, repository, digest, **kwargs):
-        # type: (str, str, **Any) -> DownloadBlobResult | None
+    def download_blob(self, repository: str, digest: str, **kwargs: Any) -> Union[DownloadBlobResult, None]:
```

Comment:
Can we please update the docstring to describe when the output would be `None`? That's a very odd result...

Reply:
Updated